{{cookiecutter.app_name}}/.github/workflows/deploy_mock_server.yml

```
@@ -0,0 +1,22 @@
+name: Deploy Mock Server
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  deploy:
+    name: Deploy server mock
```

It should be:

```suggestion
    name: Deploy mock server
```

🙈

---

Fixed in https://github.com/nimblehq/gin-templates/pull/108/commits/166892d62be9e6d0d486ffd22317707489538797